.cirrus.yml

```diff
@@ -145,10 +146,6 @@ build_task:
           else
               git reset --hard $CIRRUS_CHANGE_IN_REPO
           fi
-          # Some test operations & checks require a git "identity"
```

Comment:
For posterity: I'm assuming this was necessary in the days of `make install.tools` and should've been removed in #14016?

Reply:
I added a test commit to check this, and force-pushed.  Let's see what happens.  If it works I can squash it in, otherwise I'll drop it.

Reply:
Probably should have picked a better title for test commit.  Marked PR as draft.

Reply:
CI testing seems to function fine w/o the git identity so I stripped it out.